.env.example

```diff
@@ -2,3 +2,6 @@ DURATION_PER_SLOT=30
 MAX_SLOTS_PER_BOOKING=4
 MIN_SLOTS_PER_BOOKING=1
 MIN_SLOTS_BETWEEN_BOOKINGS=1
+
+DATABASE_URL="postgresql://postgres:1234@localhost:5433/nusc?schema=public"
```

Comment:
is this safe to be exposed?

Reply:
yup! this is just URL for the development database set up by docker compose, running on local host.